userRoutes.js

```diff
@@ -0,0 +1,53 @@
+const router = require('express').Router();
+const User = require('./userModel');
+
+//GET USERS
+router.get('/users', async (req,res)=> {
+    const user = await User.find();
+    res.send(user)
+})
+
+//GET USERS BY ID
+router.get('/:id', async (req,res)=> {
+    const user = await User.findById();
+    res.send(user);
+})
+
+
+//CREATE USER
+router.post('/createUser', async (req,res)=> {
+    const movie = await User.create({
+        name: req.body.name,
+        lastname: req.body.lastname,
+        username: req.body.username,
+        email: req.body.email,
+        password: req.body.password
+    });
+
+    res.send(movie);
```

Comment:
We need to make sure that there are no repeated users or emails. Before we create the user, query the DB and check if there's already a username or email like the one provided. If there is, don't allow it.